AppKit/CPView.j

```diff
@@ -3511,7 +3703,7 @@ var CPViewAutoresizingMaskKey       = @"CPViewAutoresizingMask",
         [self _decodeThemeObjectsWithCoder:aCoder];
 
         [self setAppearance:[aCoder decodeObjectForKey:CPViewAppearanceKey]];
-
+        
```

Comment:
Please configure your editor to trim trailing whitespace. All of your indented blank lines have useless spaces.